examples/js/binance-fetch-order-books.js

```diff
@@ -9,7 +9,7 @@ const cloudscraper = require ('cloudscraper')
 
 ;(async () => {
 
-    const exchange = new ccxt.cryptopia ({
+    const exchange = new ccxt.binance ({
```

Comment:
The problem is that `binance` does not have a `fetchOrderBooks` implementation )

Reply:
doh. i'll swap it to one that does in the morning